Composer/packages/client/src/components/WebChat/WebChatPanel.tsx

```diff
@@ -50,35 +54,50 @@ export const WebChatPanel: React.FC<WebChatPanelProps> = ({
   const webChatPanelRef = useRef<HTMLDivElement>(null);
   const [currentRestartOption, onSetRestartOption] = useState<RestartOption>(RestartOption.NewUserID);
-  const directLineErrorChannel = useRef<WebSocket>();
+  const webChatTrafficChannel = useRef<WebSocket>();
 
   useEffect(() => {
     const bootstrapChat = async () => {
       const conversationServerPort = await conversationService.setUpConversationServer();
       try {
-        directLineErrorChannel.current = new WebSocket(
-          `ws://localhost:${conversationServerPort}/ws/errors/createErrorChannel`
-        );
-        if (directLineErrorChannel.current) {
-          directLineErrorChannel.current.onmessage = (event) => {
-            const data: DirectLineLog = JSON.parse(event.data);
-            appendLogToWebChatInspector(projectId, data);
-            setTimeout(() => {
-              setActiveTabInDebugPanel('WebChatInspector');
-              setDebugPanelExpansion(true);
-            }, 300);
+        // set up Web Chat traffic listener
+        webChatTrafficChannel.current = new WebSocket(`ws://localhost:${conversationServerPort}/ws/traffic`);
+        if (webChatTrafficChannel.current) {
+          webChatTrafficChannel.current.onmessage = (event) => {
+            const data:
+              | ConversationActivityTraffic
+              | ConversationNetworkTrafficItem
+              | ConversationNetworkErrorItem = JSON.parse(event.data);
+            if (data.trafficType === 'network') {
+              appendWebChatTraffic(projectId, data);
+            }
+            if (data.trafficType === 'activity') {
+              appendWebChatTraffic(
+                projectId,
+                data.activities.map((a) => ({ activity: a, timestamp: a.timestamp, trafficType: data.trafficType }))
+              );
+            }
+            if (data.trafficType === 'networkError') {
+              appendWebChatTraffic(projectId, data);
+              setTimeout(() => {
+                setActiveTabInDebugPanel('WebChatInspector');
+                setDebugPanelExpansion(true);
+              }, 300);
+            }
           };
         }
       } catch (ex) {
         const response: AxiosResponse = ex.response;
-        const err: DirectLineLog = {
-          timestamp: getDateTimeFormatted(),
-          route: 'conversations/ws/port',
-          status: response.status,
-          logType: 'Error',
-          message: formatMessage('An error occurred connecting initializing the DirectLine server'),
+        const err: ConversationNetworkErrorItem = {
+          error: {
+            message: formatMessage('An error occurred connecting initializing the DirectLine server'),
+          },
+          request: { route: 'conversations/ws/port', method: 'GET', payload: {} },
+          response: { payload: response.data, statusCode: response.status },
+          timestamp: new Date().toISOString(),
```

Comment:
do we want toISOString or locale string?

Reply:
This is just on the server side, on the client side it gets transformed into a different format respective of the locale. I think I'm going to end up using `Intl.DateTimeFormat().format()` on the client though ([reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat))

Reply:
Fixed